src/partials/article.hbs

```diff
@@ -1,4 +1,4 @@
-<article class="doc">
+<article class="doc{{#if page.attributes.flag-attributes}} {{page.attributes.flag-attributes}}{{/if}}">
```

Comment:
I would not add additional classes on the `article`, you can already use `page-theme` and `page-role`.

Reply:
OK, the goal was to style the h1, because unlike the other heading levels it doesn't get role converted to class (being title, not part of content, I suppose). It looks like I can use the class added to the body tag (although it doesn't seem to work when using `gulp preview`)